cmake/FindV8.cmake

```diff
@@ -0,0 +1,22 @@
+if(NOT V8_FOUND)
+  if(APPLE)
+    add_library(v8 INTERFACE IMPORTED)
+    # Since the "includejs" library is compiled with the `-isysroot` flag of
+    # Clang, it is unable to find the V8 library and headers which have been
+    # downloaded from Homebrew.
+    # Refs: https://github.com/Homebrew/homebrew-core/issues/45061#issuecomment-541420664
+    set(V8_VERSION "12.1.285.24")
+    target_include_directories(v8
+      INTERFACE "/usr/local/Cellar/v8/${V8_VERSION}/include")
```

Comment:
`/usr/local/Cellar` is specific to Intel macOS. On Apple Silicon, this is `/opt/homebrew/Cellar`. I guess you can address that when adding Apple Silicon support on GitHub Actions

Reply:
Done in https://github.com/crossnx/includejs/pull/47